<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlicerTumorSegmentator</class>
 <widget class="qMRMLWidget" name="SlicerTumorSegmentator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>410</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Inputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>T1:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelector">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Pick the T1 input to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ceT1:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="cet1Selector">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Pick the ceT1 input to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist/>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string>Singleton</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>T2:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qMRMLNodeComboBox" name="t2Selector">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Pick the T2 input to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Flair:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="qMRMLNodeComboBox" name="flairSelector">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Pick the FLAIR input to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="2">
       <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton">
        <property name="text">
         <string>Outputs</string>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Output segmentation</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="qMRMLNodeComboBox" name="outputSelector">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Pick the output to the algorithm.</string>
           </property>
           <property name="nodeTypes">
            <stringlist>
             <string>vtkMRMLSegmentationNode</string>
            </stringlist>
           </property>
           <property name="showChildNodeTypes">
            <bool>false</bool>
           </property>
           <property name="noneEnabled">
            <bool>true</bool>
           </property>
           <property name="addEnabled">
            <bool>true</bool>
           </property>
           <property name="removeEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Reference:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="qMRMLNodeComboBox" name="refSelector">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Pick the Reference Volume</string>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QCheckBox" name="checkCoregistered">
        <property name="text">
         <string>Already co-registered</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QCheckBox" name="checkSkullstripped">
        <property name="text">
         <string>Already skull-stripped</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QCheckBox" name="checkDeleteProcessed">
        <property name="text">
         <string>Remove Pre-Processed Images</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="statusLabel">
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>SlicerTumorSegmentator</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>flairSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SlicerTumorSegmentator</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>outputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SlicerTumorSegmentator</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>cet1Selector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SlicerTumorSegmentator</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>t2Selector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SlicerTumorSegmentator</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SlicerTumorSegmentator</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>refSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
